src/pages/iou/steps/MoneyRequestAmount.js

```diff
@@ -167,7 +167,7 @@ const replaceAllDigits = (text, convertFn) =>
         .join('')
         .value();
 
-function MoneyRequestAmountPage(props) {
+function MoneyRequestAmount(props) {
```

Comment:
Not sure this rename is necessary, maybe would undo it to reduce the diff?

Reply:
@s77rt asked for the rename above, I am happy to do either.